src/main/java/org/mule/tools/apikit/input/RAMLFilesParser.java

```diff
@@ -54,30 +54,23 @@ private void collectResources(String filePath, Map<String, Resource> resourceMap
       for (Action action : resource.getActions().values()) {
 
         Map<String, MimeType> mimeTypes = action.getBody();
-        boolean addGenericAction = false;
         if (mimeTypes != null && !mimeTypes.isEmpty()) {
           for (MimeType mimeType : mimeTypes.values()) {
-            if (mimeType.getSchema() != null
-                || (mimeType.getFormParameters() != null && !mimeType.getFormParameters().isEmpty())) {
-              addResource(api, resource, action, mimeType.getType(), version);
-            } else {
-              addGenericAction = true;
-            }
+            boolean shouldIncludeMimeTypeInName = mimeType.getSchema() != null ||
```

Comment:
Is the same logic but _improved_

https://github.com/mulesoft/apikit-rest-scaffolder/blob/f3571560700d67ef87faf0dd38631834a6044b80/src/main/java/org/mule/tools/apikit/input/RAMLFilesParser.java#L64

When the "addGenericAction" is true, it doesn't includes the mime type in the apikit-flow name. This is a problem for MUnit because it ALWAYS includes the flowname + mimetype in the test name, so if we don't do this, we could have multiple tests with the same name.